titleToNumber.js

```diff
@@ -0,0 +1,38 @@
+const titleToNumber = (columnTitle) => {
```

Comment:
For each task lets have a "preface" comment with a link to corresponding description (of what it's about), like:
```
// This file contains solution to the following problem:
// https://some-url.some

const titleToNumber = (columnTitle) => {
```